finality-aleph/src/sync/mod.rs

```diff
@@ -124,6 +124,26 @@ pub enum BlockStatus<J: Justification> {
     Unknown,
 }
 
+/// FinalizationStatus of the block
+pub enum FinalizationStatus<J: Justification> {
+    /// The block is finalized by justification.
+    FinalizedWithJustification(J),
+    /// The block is finalized because one of its children is finalized.
+    FinalizedByChild(BlockIdFor<J>),
```

Comment:
Wouldn't it be more convenient to return a `Header` here rather than just id? For the purposes of the user I mean, I saw that in the implementation this already would be simpler. :P

Reply:
yep, done